src/sonic-host-services/scripts/sonic-host-server

```diff
@@ -13,9 +13,17 @@ import dbus.mainloop.glib
 
 from gi.repository import GObject
 
-def register_modules():
+def find_module_path():
+    """Find path for host_moduels"""
+    try:
+        from host_modules import host_service
+        return os.path.dirname(host_service.__file__)
+    except Exception as e:
+        print("error occurred in find_module_path: {}".format(sys.exc_info()[1]))
```

Comment:
>print [](http://example.com/codeflow?start=8&length=5)

Normally do not `print` inside a general function. If you really want to `print`, you may call it and check return value. So the `print` will be in `main` function.